IndicoV2.IntegrationTests/Submissions/SubmissionClientTests.cs

```diff
@@ -98,6 +98,20 @@ public async Task CreateAsync_ShouldCreateSubmission_FromUri()
             submissionId.Should().BeGreaterThan(0);
         }
 
+        [Test]
+        public async Task CreateAsync_ShouldCreateMultiFileSubmission_FromFilePath()
+        {
+            // Arrange
+            var filePath = _dataHelper.Files().GetSampleFilePath();
+
+            // Act
+            var submissionIds = await _submissionsClient.CreateAsync(_workflowId, paths: new[] { filePath, filePath }, bundle: true, resultsFileVersion: SubmissionResultsFileVersion.Three);
```

Comment:
It would help here to not use named parameters so we can catch signature changes that might be breaking

Reply:
removed named parameters except for paths